### ISSUE TITLE
resemble use in nodejs cause cache increase and not release

### ISSUE THREAD

**Issue by imdoge** [HuddleEng/Resemble.js#124]
test the code in readme in nodejs 
```
const data = await compareImages(
	await fs.readFile('./demoassets/People.jpg'),
	await fs.readFile('./demoassets/People2.jpg'),
	options,
);
```
run this code several times, cache increase unlimited

**Comment by imdoge** [comment]
can it be fixed?

**Comment by jamescryer** [comment]
I'm not sure I know what you mean by cache increase. Are you seeing an error logged in console?  Can you describe the effect, how do you monitor it?

**Comment by imdoge** [comment]
![image](https://user-images.githubusercontent.com/13634423/35145516-116e7c3e-fd43-11e7-8b32-099a4ae513dd.png)

I use pm2 list，notice that if I run the compareImages many times, the memery will increase unlimited (several GB)
I just use the basic demo code in readme, is that my problem or the problem of library?
p.s  excuse my English  : )

**Comment by imdoge** [comment]
```
router.post('/test', async function (req, res) {
  try {
    var result = await recognition()
    res.json({code: 200, result: result})
  } catch (err) {
    res.json({code: 500, result: err})
  }
})
```


```
var fs = require('fs')
var axios = require('axios')
var compareImages = require('resemblejs/compareImages')
var Canvas = require('canvas')

async function recognition() {

const data = await compareImages(
        fs.readFileSync('./img1.jpg'),
        fs.readFileSync('./img2.jpg')
);

await fs.writeFile('./output.png', data.getBuffer());

return -1;
}
module.exports = recognition;
```


the basic test code, hide other code

**Comment by imdoge** [comment]
![image](https://user-images.githubusercontent.com/13634423/35146539-4f33aece-fd46-11e7-9edf-0195d999bcde.png)
post this api dozens of times,  find the memery increase

**Comment by imdoge** [comment]
thx

**Comment by jamescryer** [comment]
I believe this is fixed now, as of v2.10.0
